xml/System.Threading/CancellationToken.xml

```diff
@@ -1013,6 +1013,7 @@ End If
         <related type="Article" href="/dotnet/standard/threading/cancellation-in-managed-threads">Cancellation</related>
         <related type="Article" href="/dotnet/standard/parallel-programming/task-cancellation">Task Cancellation</related>
         <related type="Article" href="/dotnet/standard/parallel-programming/how-to-cancel-a-task-and-its-children">How to: Cancel a Task and Its Children</related>
+        <exception cref="T:System.ObjectDisposedException">The associated <see cref="T:System.Threading.CancellationTokenSource">CancellationTokenSource</see> has been disposed.</exception>
```

Comment:
This API doesn't actually throw `ObjectDisposedException` and that was removed in the docs in https://github.com/dotnet/dotnet-api-docs/pull/7555. I think the XML comment should also be removed.

Reply:
Got it. Then it should also be removed [from source](https://github.com/dotnet/runtime/blob/853e494abd1c1e12d28a76829b4680af7f694afa/src/libraries/System.Private.CoreLib/src/System/Threading/CancellationToken.cs#L354), otherwise the porting tool will keep detecting it and porting it on subsequent previews.

I created this issue to track the request: https://github.com/dotnet/runtime/issues/66369